docs/index.md

```diff
@@ -3,19 +3,19 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# UGTeach
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
Can help change URLs to these:
1st: https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.sv
2nd: https://github.com/AY2425S1-CS2103T-F14a-1/tp/actions

Reply:
Thanks for catching this! Ive made the changes.

Reply:
Sorry I made a mistake for the first link. Should be this:
https://github.com/AY2425S1-CS2103T-F14a-1/tp/workflows/Java%20CI/badge.svg

Reply:
Thanks for the correction!